chore(scripts): turn various docs items into scripts

Langleu opened the pull request:
- core-dns config generation
- core-dns test
- elasticsearch secrete creation
- environment variable export
- zeebe helm vars from python to bash

maxdanilov commented on create_elasticsearch_secrets.sh:
```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```
I suggest either explicitly naming this file `create_elasticsearch_secrets_aws.sh` or similar, or expanding it to the point that we can support more cloud providers in the future and it's not S3-specific.

maxdanilov replied:
e.g. script can be called as `create_elasticsearch_secrets.sh aws` or `create_elasticsearch_secrets.sh gcp` (with anything other than `aws` for now throws a "not implemented" error)

Langleu replied:
I'd go for now with the explicit file naming since maybe there's other implementations required etc. etc. so rather redo the docs later and unify the scripts than spend now the time for something that may never happen.

Langleu replied:
It's part of the subfolder `aws/dual-region/scripts`, which I think should be clear that's provider specific.
If and when we add more providers, we can potentially abstract logic into shared code functions.

maxdanilov commented on create_elasticsearch_secrets.sh:
```diff
@@ -0,0 +1,43 @@
+        --from-literal=S3_SECRET_KEY="$secret_access_key"
+}
+
+if [ -z "$ACCESS_KEY" ]; then
```
suggestion: use `AWS_ACCESS_KEY` and `AWS_SECRET_ACCESS_KEY` to explicitly show this is only for AWS installations.

Langleu replied:
changed and added a suffix `_ES` as otherwise it will potentially clash with the AWS CLI config and cause more harm than it does good.

maxdanilov commented on export_environment_preqrequisites.sh:
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
```
how is this meant to be called?

trying

```
./export_environment_preqrequisites.sh
```
and then
```
echo $REGION_0
```
but it's empty

maxdanilov replied:
guess it should be 

```
source ./export_environment_preqrequisites.sh
```
?

maxdanilov replied:
nvm, saw the answer in https://github.com/camunda/camunda-docs/pull/3430 with `. <script-name>`

maxdanilov commented on generate_core_dns_entry.sh:
```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
```
should this contain 
```bash
set -e
```
?

(and same for the other scripts)

maxdanilov commented on generate_zeebe_helm_values.sh:
```diff
@@ -0,0 +1,72 @@
+elastic1=$(generate_exporter_elasticsearch_url "$namespace_1" "$helm_release_name")
+
+echo
+echo "Please use the following to set the environment variable ZEEBE_BROKER_CLUSTER_INITIALCONTACTPOINTS in the base Camunda Helm chart values file for Zeebe."
```
shall we also clarify that it does into the `zeebe.env` path of the shared base yaml overlay?

maxdanilov commented on test_dns_chaining.sh:
```diff
@@ -0,0 +1,47 @@
+create_namespace "$CLUSTER_0" "$CAMUNDA_NAMESPACE_0"
+create_namespace "$CLUSTER_1" "$CAMUNDA_NAMESPACE_1"
+
+kubectl --context "$CLUSTER_0" apply -f https://raw.githubusercontent.com/camunda/c8-multi-region/main/test/resources/aws/2-region/kubernetes/nginx.yml -n "$CAMUNDA_NAMESPACE_0"
```
`/test/` path will be adjusted later?

maxdanilov reviewed (changes requested):
Thank you, looks good overal, leaving minor comments.

Suggestion: move all these scripts into a separate folder (`dual-region` or similar), so in the future it's easier to separate concerns in the same repo for 2 vs 3-region setups.

maxdanilov commented on generate_core_dns_entry.sh:
```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
```
same here, I suggest showing in the name that this script is only for AWS or implement it in a way more providers to be supported in the future.

Langleu replied:
same as above.

Langleu commented:
thanks for the review!
I moved the folder in a different PR but also renamed now the `2-region` to `dual-region` to align with the same overall style.
Also adjusted the docs to mention the folder path compared to previously `root`.

maxdanilov reviewed (approved):
all good, just last file rename missing

maxdanilov commented on aws/dual-region/scripts/generate_core_dns_entry.sh:
```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
```
nvm, `aws` is present in the folder name.
